src/app/_components/home.tsx

```
@@ -2,99 +2,548 @@
+  const { isSignedIn } = useAuth();
+  if (!personaId) return null;
+
+  if (!isSignedIn) return <p>Sign in to enhance your persona</p>;
+
+  const generate = async () => {
+    const { object, personaEventId } = await enhancePersonaAction(
+      personaId,
+      prompt
+    ).catch((error) => {
+      console.log("error", error);
+      if (error.message && error.message.includes("Insufficient tokens")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)